# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/nsICookieManager.idl
# Commit: 98848e63dfbf
# Full Hash: 98848e63dfbf5702726d3e07fb92855263312fed
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r b668d0c685e8 -r 98848e63dfbf netwerk/cookie/nsICookieManager.idl
--- a/netwerk/cookie/nsICookieManager.idl	Wed Jan 08 19:25:59 2025 +0000
+++ b/netwerk/cookie/nsICookieManager.idl	Wed Jan 08 19:26:00 2025 +0000
@@ -311,4 +311,7 @@
    */
   void removeThirdPartyCookieBlockingExceptions(
     in Array<nsIThirdPartyCookieExceptionEntry> aExceptions);
+
+  // Test getter to inspect remote exception list state.
+  Array<ACString> testGet3PCBExceptions();
 };